packages/lab/src/dropdown-next/DropdownNext.css

```diff
@@ -0,0 +1,111 @@
+.saltDropdownNext-button {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor);
+  --dropdownNext-borderStyle: var(--salt-editable-borderStyle);
+  --dropdownNext-outlineColor: var(--salt-focused-outlineColor);
+  --dropdownNext-borderWidth: var(--salt-size-border);
+  --dropdownNext-padding: var(--salt-size-unit);
```

Comment:
```suggestion
  --dropdownNext-padding: var(--salt-spacing-100);
```

Reply:
would say this isnt needed tbh, just use spacing-100 directly on the padding since its not redefined